VirtoCommerce.Platform.Web/Modularity/ManifestModuleCatalog.cs

```diff
@@ -170,25 +170,50 @@ public override void Validate()
             return result;
         }
 
-        private static void CopyAssemblies(string sourceParentPath, string targetDirectoryPath)
+        private static void CopyLocalizationAssemblies(string sourceDirectoryPath, string targetDirectoryPath)
+        {
+            var cultureNames = ConfigurationHelper.GetAppSettingsValue("VirtoCommerce:Resources:CultureNames") ?? string.Empty;
+
```

Comment:
Please make a lazy property for this option to read it only once.

Reply:
This app setting removed